podcasts/TranscriptManager.swift

```diff
@@ -78,6 +78,11 @@ class TranscriptManager {
             throw TranscriptError.failedToLoad
         }
 
+        let trimmedTranscript = transcriptText.trimmingCharacters(in: .whitespacesAndNewlines)
```

Comment:
what about adding this check inside the TranscriptModel.makeModel method a bit bellow?

Just thinking that this is more `failedToParse` error than a `failedToLoad`